### ISSUE TITLE
Suggestion: Add missing episode dropdown list with search?

### ISSUE THREAD

**Issue by kolbdog323** [theMK2k/Media-Hoarder#72]
It might be useful if you dont know the season and episode number of the episode you wish to add to media hoarder and dont want to go searching for the correct season and episode number on imdb or google sicne sometiems you cant find the episode simply by google searching imdb then episode name or tv show name.

**Comment by theMK2k** [comment]
Hm, I wonder what the workflow with this might be.

Imagine we have Bluey added a series. And a new episode comes around, for example "The Sign" (https://www.imdb.com/title/tt31491044/)

Without knowing the season and episode numbers, you might have this as `Bluey - The Sign.mkv`, correct?

You then add this to the already available `Bluey` series folder and do a rescan with Media Hoarder.

What then?

I see two scenarios:

- Media Hoarder's IMDB detection gets more intelligent by finding the correct episode also by just the name in the filename (no manual search is necessary)
- Media Hoarder's "Link with IMDB entry" dialog allows to search within the series' episode list that Media Hoarder fetches from IMDB first

Both scenarios will be eventually implemented in Media Hoarder. Definitely before the tv series feature leaves the beta stage.

If there's anything else, I miss, please let me know.

**Comment by kolbdog323** [comment]
Not what i meant i mean it shows all the episodese from imdb in a drop down list like right click the tv show name and can see missing episodes or soemthing like that so before you add the video to the library you can see what episdoes are missing and then get the season and epsidoes numbers and properly name the file before you add it.

**Comment by theMK2k** [comment]
Ah, thanks for the clarification. That's an interesting feature. It will also help in tracking the series' completion status.

**Comment by kolbdog323** [comment]
and maybe a search function for that so you can type in the name of the episode your looking to get the proper season and episdoe number for?

**Comment by theMK2k** [comment]
Absolutely!

**Comment by kolbdog323** [comment]
Glad you approve of this suggestion. Well I hope to see it implemented sometime in the future.

**Comment by kolbdog323** [comment]
And maybe add a small tag to the show that says in progress or completed so you know if you have all the episodes in the show or not.

**Comment by kolbdog323** [comment]
W

> And maybe add a small tag to the show that says in progress or completed so you know if you have all the episodes in the show or not. In the title of the show somewhere or before it and not where all the extra info is.

What do you think about this little suggestion abotu inprogress or completed or maybe show the ammount of missing episodes?

**Comment by theMK2k** [comment]
Yes, something like this will also come, as well as filters to quickly see series that are incomplete or complete by that measure.
